cime_config/testdefs/testlist_clm.xml

```diff
@@ -1333,7 +1333,7 @@
       <machine name="derecho" compiler="intel" category="aux_clm"/>
     </machines>
     <options>
-      <option name="wallclock">01:40:00</option>
+      <option name="wallclock">01:40:01</option>
```

Comment:
In case this comes in after the b4b-dev merge (which it probably will) that changes this test to 2:30:00.